trusted-types/support/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.js

```diff
@@ -48,32 +52,38 @@ test(t => {
   });
 }, `\`${globalThisStr}.setInterval(null)\` throws.`);
 
+promise_test(async t => {
+  const input = ";".repeat(100);
+  let violation = await trusted_type_violation_for(TypeError, _ =>
+    globalThis.setTimeout(input)
+  );
+  assert_equals(violation.blockedURI, "trusted-types-sink");
+  assert_equals(violation.sample, `${expectedSink} setTimeout|${clipSampleIfNeeded(input)}`);
+}, `Violation report for ${globalThisStr}.setTimeout`);
```

Comment:
So just for the record, my idea was:
- to have something like https://github.com/web-platform-tests/wpt/pull/50276/files#diff-73726b09a36e4a52b58b2a264fa5ee6006ead6f25fab1eb019aeffb7329d7cea testing TrustedScript does not cause violation and plain string does cause violation.
- test setInterval too.

of course in the draft I shared with you I was just checking one case since my problem was mainly to make the test works in workers...